### PR TITLE
Basic setup of the leader board app

### PR DESCRIPTION
**Pull request by veronica365**
### Description

- Basic setup of the app with minimal CSS styles and simple Javascript functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,144 @@
...
+
+- **Has a page with a list of scores**
+- **Has a section where a user can add a new score**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://veronica365.github.io/leaderboard-app/dist/)
+
```
Comment (rotimiazeez) [inline README.md:53]: - Kindly add a valid demo link to your live demo, or add coming soon... if you don't have one yet 😉.

File: README.md [hunk 2]
```diff
@@ -1 +1,144 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rotimiazeez) [inline README.md]: - Please provide a valid MIT license link to your README; the current link is broken. 👍🏽 
Screenshot from 2023-03-08 17-39-38

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @veronica365 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️
- Good project setup 👏🏽
- Good UI 👍🏽 
- Good commit message(s) 😉
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (ShadyShawkat) [inline README.md]: - [ ] As requested by the previous review, kindly make sure that the link to the MIT license is working properly. Currently, this page shows when I click on the MIT link.
image

- [HINT] The issue here lies in a typo in the name of the file.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @veronica365, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: LICENCE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
+
```
Comment (tresorsawasawa) [inline LICENCE]: - [ **OPTIONAL** ] I think it would be better if you add the `Year` and your `full name` at the showed `placeholder` above.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @veronica365 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**